elearning-frontend/src/pages/quizzes/QuizPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import getWords from '../../services/actions/getWords'
+import ReactPaginate from 'react-paginate'
+import QuizItem from './QuizItem'
+import Divider from '../../core/Divider'
+
+const QuizPage = () => {
+    let { slug } = useParams()
+
+    const [changeWordData, setChangeWordData] = useState(true)
+    const [wordList, setWordList] = useState([])
+
+    const [currentItem, setCurrentItem] = useState(null)
+    const [pageCount, setPageCount] = useState(0)
+    const [itemOffset, setItemOffset] = useState(0)
+
+    // Fetch all words
+    const getWordList = () => {
+        getWords(setWordList, slug)
+        setChangeWordData(false)
+    }
+
+    useEffect(() => {
+        const fetchWords = async() => {
+            const data = getWordList()
+        }
+
+        fetchWords()
```

Comment:
```suggestion

     getWordList()
```
your function is calling another function. you can just call the called function directly. then you set the data into another variable which does nothing. You add up some memory but you did nothing which will cause some slowness in the future if you have so many of this.